docs/meshcloud.policies.md

```diff
@@ -75,18 +78,19 @@ Your organization is fully free to define policies across the entire meshStack.
 
 ### How are meshPolicies evaluated
 
-As mentioned before, meshPolicies are built on top of meshStack's [tagging](./meshcloud.metadata-tags.md) system. This is done by evaluating two meshObjects on their tags. There is no "direction" in a meshPolicy: the meshObjects are evaluated bi-directionally. It does not matter if you have a meshPolicy for "meshCustomer & meshProject" or "meshProject & meshCustomer", the policy evaluation is equal.
-If a policy is defined, the two meshObjects' tags will be evaluated on a so-called set intersection. This means that for either tags, at least one value must be present on both sides for a successful evaluation. If neither side have the given tag, the policy is also successful.
+As mentioned before, meshPolicies are built on top of meshStack's [tagging](./meshcloud.metadata-tags.md) system. This is done by evaluating two meshObjects on their tags. There is the "direction" between the affected meshObject and authoritative meshObject in a meshPolicy: the meshObjects are evaluated unidirectionally.
```

Comment:
```suggestion
As mentioned before, meshPolicies are built on top of meshStack's [tagging](./meshcloud.metadata-tags.md) system. This is done by evaluating two meshObjects on their tags. There is the "direction" between the affected meshObject and authoritative meshObject in a meshPolicy: the meshObjects are evaluated unidirectionally.
```
This part of the documentation needs to be rewritten when we change the policy dialog in the meshPanel